internal/controller/gitclient.go

```diff
@@ -0,0 +1,81 @@
+package controller
```

Comment:
This file can live in a different package, such as `git`, so it can be imported as `git.IssueClient.List(...)`